src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/enums/AssigneeEnumTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.enums;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.TaskState;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class AssigneeEnumTest {
+
+    @Test
```

Comment:
I mean having a test for this isn't bad but perhaps let's also add another test to assert the size of TaskState that way if someone adds something he'll need to remember to update the test.

also TaskState.ASSIGNED.getTaskState() should be toString()